manifests/repository.pp

```diff
@@ -5,10 +5,22 @@
 # This class will be automatically included when a resource is defined.
 # It is not intended to be used directly by external resources like node definitions or other modules.
 class bareos::repository (
-  $release = 'latest',
-  $gpg_key_fingerprint = undef,
+  String           $release             = 'latest',
+  Optional[String] $gpg_key_fingerprint = undef,
```

Comment:
suggest `Optional[String[1]]`

Reply:
done
whats the purpose of this syntax? I can't find an explanation.

Reply:
it defaults to undef, if a string is passed, the minimal length is 1. this basically prohibits passing `''` (empty string)